server/src/controllers/GrailController.ts

```diff
@@ -86,8 +102,157 @@ export class GrailController {
       dataToSet.runewordData = runewordGrailData;
       return dataToSet;
     });
+
+    this.getMemberParties(address, parties => {
+      parties.forEach(party => {
+        this.saveGrailInPartyDatabase(address, party.address, grailData);
+      });
+    });
   };
 
+  public saveGrailInPartyDatabase = async (
+    grailAddress: string,
+    partyAddress: string,
+    data: any
+  ) => {
+    let partyUserData = GrailController.formatGrailForParty(data);
+    const result = await this.partyCollection.findOneAndUpdate(
+      {
+        address: GrailController.trimAndToLower(partyAddress),
+        "data.users.username": grailAddress
+      },
+      {
+        $set: {
+          "data.users.$.data": partyUserData
+        },
+        $inc: { updateCount: 1 }
```

Comment:
I don't think you need the updateCount here

Reply:
This code is removed on latest commit